resources/migrations/001_update_migrations.yaml

```diff
@@ -5652,7 +5652,151 @@ databaseChangeLog:
             onDelete: CASCADE
 
   - changeSet:
-      id: v50.2024-01-10T03:27:30
+      id: v50.2024-02-20T19:21:04
+      author: camsaul
+      comment: Drop v1 version of v_content view since it references report_card.dataset which we are dropping in next migration
```

Comment:
I think it's actually an artifact of the diff. I had to add changeSets before the permissions migrations, which meant I had to change the IDs for the permission migration changeSets (so they'd be in order), which meant all the migrations now come after Cam's migration instead of before. The diff thinks I've added Cam's migration up here and deleted it [here](https://github.com/metabase/metabase/pull/39170/files#diff-e6ec4fc8084d80ae2ebedde98c98f358ec7ac526de5c339ebcb454cbda62b221L5758-L5825).

Basically, before things looked like:
- Permission migration (id 1)
- Cam's change (id 2)

I had to change this to:
- Cam's change (id 2)
- My new perms changes (id 3)
- Permission migration (id 4)

It actually occurs to me that before we merge this branch we will need to reorder all these migrations again, to make sure that all the new migrations come after any migrations already in master - e.g. the migration that creates the `data_permissions` table will need to get placed after Cam's changeSet. So these will get all messed up again 😂

Reply:
Hmm I don't know if migrations necessarily need to go into `master` in order. I've definitely merged migrations in that had other migrations after it...especially when we were reserving migration IDs ahead of time.

Reply:
I might be wrong, but I thought I'd gotten lint errors before for this, and it looks like the linter at least is intended to block out-of-order migrations.

Do you want me to try without re-arranging, or is this ok as-is?

Reply:
The IDs in the migration file need to be in order, but they don't need to be merged into master in order, necessarily.

Reply:
Stepping back a bit — you were saying that you had to add migrations before the perm migrations, which meant that all of them had to come after Cam's migrations.

But could you add the new migrations to have lower IDs than the existing perm migrations? Then you wouldn't have to move them at all. Especially since we're using timestamps now and not reserving IDs.

Reply:
So the risk I was thinking about here was:
- Cam merges Change 2
- A point release goes out to cloud - it's migrated to 2
- We merge the feature branch with Change 1
- Migration fails because you can't run Change 1 after change 2 has already run.

But this was just my assumption, and apparently... it's wrong! You can run `migrate!` with change 2, then add in change 1 and `migrate!` again and things work just fine. Which is honestly confusing and a little scary to me, because then migrations can run in an unpredictable order - but I guess this is why we try to only include migrations in major releases. 😓 

Thanks for walking me through this until I finally tested my wrong assumptions!

Reply:
Yeah that's a pretty reasonable assumption to make. I guess as long as all related migrations go in at the same time they'll be run in the same order, but it might be after other migrations with later IDs have already been run. Not sure if we've hit any issues because of that before.

In this case I think we're fine though since all of this should go out in 50, not in any minor releases before then. i.e. we'll merge to `master` but not backport to the 49 release branch.